### PR TITLE
docs: contributing link fix

### PR DESCRIPTION
**Pull request by vinckr**
There is still an error in
Line 163 the link to /docs/README.md
is broken (which I put in there btw.)

I tried to use (../../README.md) to make it work, but it does not and I dont know why.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -157,7 +158,7 @@ should be merged by the submitter after review.
 ## Documentation
 
 Please provide documentation when changing, removing, or adding features.
-Documentation resides in the project's [docs](docs) folder. Generate API and
+Documentation resides in the project's docs folder. Generate API and
 configuration reference documentation using `cd docs; npm run gen`.
 
 For further instructions please head over to [docs/README.md](docs/README.md).
```
Comment (vinckr) [inline CONTRIBUTING.md:164]: This is broken, how can we fix this that it works with the script?
Or just leave out the link?

**Comment by vinckr** [comment]
@zepatrik 
I would do the sidebar changes, when this PR is good. 
I dont know why the cla check fails :/

**Comment by zepatrik** [comment]
`../../README.md` is outside of the docusaurus project so it can not be resolved.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -157,10 +158,11 @@ should be merged by the submitter after review.
 ## Documentation
 
 Please provide documentation when changing, removing, or adding features.
-Documentation resides in the project's [docs](docs) folder. Generate API and
+Documentation resides in the project's docs folder. Generate API and
```
Comment (zepatrik) [inline CONTRIBUTING.md:161]:
```suggestion
Documentation resides in the project's `docs` folder. Generate API and
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -157,10 +158,11 @@ should be merged by the submitter after review.
...
 configuration reference documentation using `cd docs; npm run gen`.
 
-For further instructions please head over to [docs/README.md](docs/README.md).
+For further instructions please head over to
+[docs/README.md](ttps://github.com/ory/{{Project}}/blob/master/README.md).
```
Comment (zepatrik) [inline CONTRIBUTING.md:165]:
```suggestion
[docs/README.md](ttps://github.com/ory/{{Project}}/blob/master/docs/README.md).
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -77,7 +78,7 @@ do this is via the [ORY Community](https://community.ory.sh/) or join the
 ## How can I contribute?
 
 If you want to start contributing code right away, we have a
-[list of good first issues](https://github.com/ory/{{Project}}/labels/good%20first%20issue).
+[list of good first issues](https://github.com/ory/{{Project}/labels/good%20first%20issue).
```
Comment (zepatrik) [inline CONTRIBUTING.md:81]: This looks wrong
```suggestion
[list of good first issues](https://github.com/ory/{{Project}}/labels/good%20first%20issue).
```

**Review by zepatrik — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -157,10 +158,11 @@ should be merged by the submitter after review.
 ## Documentation
 
 Please provide documentation when changing, removing, or adding features.
-Documentation resides in the project's [docs](docs) folder. Generate API and
+Documentation resides in the project's `docs` folder. Generate API and
```
Comment (aeneasr) [inline CONTRIBUTING.md:161]: Why not link this like you have done below?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -13,17 +13,18 @@ https://github.com/ory/meta/blob/master/templates/repository/CONTRIBUTING.md
...
-- [How can I contribute?](#how-can-i-contribute)
-- [Communication](#communication)
-- [Contributing Code](#contributing-code)
-- [Documentation](#documentation)
-- [Disclosing vulnerabilities](#disclosing-vulnerabilities)
-- [Code Style](#code-style)
-- [Pull request procedure](#pull-request-procedure)
-- [Working with Forks](#working-with-forks)
-- [Conduct](#conduct)
+- [Contributing to ORY {{Project}}](#contributing-to-ory-project)
```
Comment (aeneasr) [inline CONTRIBUTING.md:16]: Please revert this

**Comment by vinckr** [comment]
hm when i `npm run format` 
it always adds 
- [Contributing to ORY {{Project}}](#contributing-to-ory-project)
to the TOC. 
when I dont do it, the format check fails :/.

**Comment by zepatrik** [comment]
Does not do that for me :shrug:

**Comment by aeneasr** [comment]
@vinckr this is the wrong file. The changes need to be done here: https://github.com/ory/meta/blob/master/templates/repository/common/CONTRIBUTING.md

**Pull request merged**